daisy_rani_kesavan/week_02/day02/main.html

```diff
@@ -0,0 +1,100 @@
+<html>
+<head>
+	<title>
+		Famous computer scientist fansite
+	</title>
+	</head>
+	<body>
+
+
+<div class="header">
+<div class="container">
+<img class="left" src="http://i.imgur.com/o2zbyru.jpg"/>
+</div>
+<div class="container1">
+<img class="left" src="http://i.imgur.com/zpwaKij.jpg"/>
+</div>
+<div class="container2">
+<img class="left" src="http://i.imgur.com/bxjrdSq.jpg"/>
+</div>
+</div>
+
+
+<div class="box">
+<p>Here we have a lot of scientists</p>
+<ul>
+<li>Ada Lovelace</li>
+<li>Alan Turing</li>
+<li>John von Neumann</li>
+<li>Norbert Wiener</li>
+<li>Claude Shannon</li>
+<li>Grace Hopper</li>
+<li>H. Marshall Mcluhan</li>
+<li>Donald Knuth</li>
+<li>Richard M. Stallman</li>
+<li>Larry Wall</li>
+<li>Yukihiro Matsumoto</li>
+<li>Jenn Schiffer</li>
+</ul>
+</div>
+
+<div class="box1">
+<h1>This is about <a href="biography.html">Grace Hopper</a></h1>
+<figure>
+<img src="http://i.imgur.com/ljQz4M9.jpg"/>
+<figcaption>Rear Admiral Grace M. Hopper,<br> USN, Ph.D.</figcaption>
+</figure>
+</div>
+
+
+<style>
+
+.header{
+	float:left;
+	background-color: skyblue;
+	
+    }
+
+.image123{
+	float:left;
+}
+
+.container{
+	float:left;
+}
+
+.container1{
+	float:left;
+	margin:0 5 5 5;
+}
+
+.container2{
+	float:left;
+	margin:0 5 5 5;
+}
+.left{
+	float:left;
+	width: 300px;
+    height: 300px;
+}
+
+figure{
+	text-align: center;
+	margin:0 0 0 0;
```

Comment:
margin: 0 0 0 0; you can replace with margin: 0; - as you have 0 from all sides.
Originally it goes from top clockwise. margin: top, right, bottom, left;
If you have same margin on the top and bottom, right and left you can give only 2 values as so: margin: 10px 5px; - means top and bottom are margin on 10px, right and left margin on 5.